src/Css.elm

```diff
@@ -6448,6 +6465,685 @@ outset =
 
 
 
+{- BORDER RADIUS -}
+
+
+{-| Sets [`border-radius`](https://css-tricks.com/almanac/properties/b/border-radius/) property.
+
+    borderRadius  (em 4)
+    borderRadius2 (em 4) (px 2)
+    borderRadius3 (em 4) (px 2) (pct 5)
+    borderRadius4 (em 4) (px 2) (pct 5) (px 3)
+
+**Note:** An implementation of shorthand property with supporting of elipse shape looks ugly,
+so if you wanna make somehting like this `border-radius: 4px 3px 6px / 2px 4px` by `elm-css`
+you should create a batch of `Style`:
+
+    [ borderTopLeftRadius     (px 4) (px 2)
+    , borderTopRightRadius    (px 3) (px 4)
+    , borderBottomRightRadius (px 6) (px 2)
+    , borderBottomLeftRadius  (px 3) (px 4)
+    ]
+
+-}
+borderRadius :
+    Value
+        { ch : Supported
+        , cm : Supported
+        , em : Supported
+        , ex : Supported
+        , inches : Supported
+        , mm : Supported
+        , pc : Supported
+        , pt : Supported
+        , pct : Supported
+        , px : Supported
+        , rem : Supported
+        , vh : Supported
+        , vmax : Supported
+        , vmin : Supported
+        , vw : Supported
+        , zero : Supported
+        , inherit : Supported
+        , initial : Supported
+        , unset : Supported
+        }
+    -> Style
+borderRadius (Value radius) =
+    AppendProperty ("border-radius:" ++ radius)
+
+
+{-| Sets [`border-radius`](https://css-tricks.com/almanac/properties/b/border-radius/) property.
+
+    borderRadius  (em 4)
+    borderRadius2 (em 4) (px 2)
+    borderRadius3 (em 4) (px 2) (pct 5)
+    borderRadius4 (em 4) (px 2) (pct 5) (px 3)
+
+**Note:** An implementation of shorthand property with supporting of elipse shape looks ugly,
+so if you wanna make somehting like this `border-radius: 4px 3px 6px / 2px 4px` by `elm-css`
+you should create a batch of `Style`:
+
+    [ borderTopLeftRadius     (px 4) (px 2)
+    , borderTopRightRadius    (px 3) (px 4)
+    , borderBottomRightRadius (px 6) (px 2)
+    , borderBottomLeftRadius  (px 3) (px 4)
+    ]
+
+**Note:** As with any shorthand property, individual sub-properties cannot inherit,
+such as in `borderRadius2 (px 1) inherit`, which would partially override existing definitions.
+Instead, the individual longhand properties have to be used.
```

Comment:
There is a limitation which I had never known (and I'm ashamed about it). I'm not sure that we should encapsulate things like this but the point is easy to do. So I've removed `inherit: Supported` for each of `borderRadius{2,3,4}` and `border{TopLeft,TopRight,BotomRight,BottomLeft}Radius2` and keep for same single-argument properties. What do you thing about it? Should we handle the limitation here or developers should do it self?

Reply:
Oh wow, and it's true of `initial` and `unset` too! 😮 

Great catch @owanturist! I think we should definitely change the other properties to reflect this, and I updated the contributing docs to note this: https://github.com/rtfeldman/elm-css/wiki/Phantom-Types:-Contributing

Reply:
I didn't check it for `initial` and `unset` but you're right! I think we should reflect this.